servers/rendering/rendering_device_graph.cpp

```diff
@@ -1517,6 +1626,24 @@ void RenderingDeviceGraph::begin() {
 #endif
 }
 
+void RenderingDeviceGraph::add_acceleration_structure_build(RDD::AccelerationStructureID p_acceleration_structure, ResourceTracker *p_dst_tracker, const Vector<ResourceTracker *> &p_src_trackers) {
+	int32_t command_index;
+	RecordedAccelerationStructureBuildCommand *command = static_cast<RecordedAccelerationStructureBuildCommand *>(_allocate_command(sizeof(RecordedAccelerationStructureBuildCommand), command_index));
+	command->type = RecordedCommand::TYPE_ACCELERATION_STRUCTURE_BUILD;
+	command->self_stages = RDD::PIPELINE_STAGE_ACCELERATION_STRUCTURE_BUILD_BIT;
+	command->acceleration_structure = p_acceleration_structure;
+
+	Vector<ResourceTracker *> trackers = { p_dst_tracker };
+	trackers.append_array(p_src_trackers);
+
+	Vector<ResourceUsage> usages = { RESOURCE_USAGE_ACCELERATION_STRUCTURE_READ_WRITE };
+	for (uint32_t i = 0; i < p_src_trackers.size(); ++i) {
+		usages.append(RESOURCE_USAGE_ACCELERATION_STRUCTURE_BUILD_INPUT);
+	}
```

Comment:
Given `add_acceleration_structure_build` will likely be called per frame, you should avoid the allocations using Vector<> here and instead use a thread_local LocalVector that gets resized to the correct size and filled in each frame instead.